lm_eval/base.py

```diff
@@ -891,14 +894,19 @@ def fewshot_context(
             # for justification of this separator.
             example_separator = "\n###\n"
 
+            labeled_examples_list = []
+            fewshottargetidx = []
+            for doc in fewshotex:
+                text = self.doc_to_text(doc)
+                targets = self.doc_to_target(doc)
+                target_idx = random.randint(0, len(targets) - 1)
+                target = targets[target_idx].strip()
+                # TODO(Jon): Given that target is now a list, should we add a space here? Anywhere else?
```

Comment:
@jon-tow 
I know previously there was a space intentionally added to the targets. We're adding one here now -- was there another place to add it?

Reply:
Nope. This looks good!